standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/MetaStoreDirectSql.java

```diff
@@ -1422,29 +1420,34 @@ public void visit(LeafNode node) throws MetaException {
         return;
       }
 
+      String nodeValue0 = "?";
```

Comment:
what is  `nodeValue0` vs `nodeValue`? can we come up with a better name.

Reply:
The `nodeValue0` is placeholder, `nodeValue` is designated parameter for the PrepareStatement, we've already used it elsewhere in this method...